src/components/HomepageFeatures/index.js

```diff
@@ -24,7 +24,18 @@ const FeatureList = [
           ChromeOS or alternative platforms? Get the web version on any browser-enabled device.
       </>
     ),
-  }
+  },
+  {
+    title: 'Lightweight & Fast',
+    Svg: require('@site/static/img/fastRocket.svg').default,
+    description: (
+      <>
+          Experience lightning-fast coding without compromise whether it be on the web or the computer. 
```

Comment:
Shorten to : 
` Experience lightning-fast coding on any device - whether it be on a web browser or our desktop apps.`

Users wont read longer lines. So we try to keep things very short and to the point in the docs.